pkg/volume/glusterfs/glusterfs.go

```diff
@@ -492,9 +513,43 @@ func (r *glusterfsVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 		return nil, err
 	}
 	r.provisioningConfig = *cfg
-
 	glog.V(4).Infof("glusterfs: creating volume with configuration %+v", r.provisioningConfig)
-	glusterfs, sizeGB, err := r.CreateVolume()
+	scName := r.options.PVC.Annotations["volume.beta.kubernetes.io/storage-class"]
+	
+	// TODO :Release GIDs
+	if startGid64, err := strconv.ParseUint(r.provisioningConfig.gidMin, 10, 32); err == nil {
+		blockStart32 = uint32(startGid64)
+		glog.V(4).Infof("glusterfs: blockstart details %#v", blockStart32)
```

Comment:
`blockstart details` looks weird to me and there already is `provisioning config and gid range %#v [%#v - %#v]` below, IMO you can delete this log (+the one for `blockend details`).